test/unit/step-collection-test.js

```diff
@@ -20,20 +21,20 @@ describe('Steps', function() {
     this.run = (done) => {
       runCount += 1;
       done();
-    }
-  }
+    };
+  };
 
   beforeEach(function() {
     this.sinon = sinon.sandbox.create();
     runCount      = 0;
-    Steps.Step    = Step;
+    StepCollection.Step = Step;
     mockStream    = new OutputStream();
     outputStream  = mockStream.stream;
     logger        = new Logger(outputStream);
     compiledSteps = compiledFeature.document.feature.children[0].steps;
     world         = new World({
       logger: logger,
-      stepDefinitions: stepDefinitions
+      stepDefinitions: StepDefinitionCollection
```

Comment:
I think there is a funky bug here on both the original and the new one. So the `stepDefinitions` that are in the world are an instance on the `StepDefinitionCollection`, not the class. I don't ever see that instance on the original. So, confused about how it was passing :(